services/ui-auth/serverless.yml

```diff
@@ -77,6 +84,52 @@ resources:
         ExplicitAuthFlows:
           - ADMIN_NO_SRP_AUTH
         GenerateSecret: false
+        AllowedOAuthFlows:
+          - implicit
```

Comment:
Eventually updating this to the `code` flow would help people get started with best practices. A bit more about it can be found [here](https://oauth.net/2/grant-types/implicit/)

Reply:
Added an enhancement issue for this.  Thanks David
#204